rust/osm2lanes/src/lib.rs

```diff
@@ -20,4 +20,5 @@ pub mod overpass;
 pub mod transform;
 pub use transform::{lanes_to_tags, tags_to_lanes, LanesToTagsConfig, TagsToLanesConfig};
 
-mod test;
+#[cfg(feature = "overpass")]
+pub mod test;
```

Comment:
ignoring the typo, I think this is the most idiomatic way to share test interfaces?

```suggestion
#[cfg(feature = "tests")]
pub mod test;
```